packages/redux/src/products/actions/factories/fetchProductListingFactory.ts

```diff
@@ -41,6 +41,7 @@ const fetchProductListingFactory =
       getState,
       options,
       false,
+      false,
```

Comment:
This is not necessary if you make the isCustomListingPage parameter an optional one with a default value of false in fetchProductListFactory.

Reply:
I follow the same in the `isSet`, since it is also optional. Did it make sense?

Reply:
Ok, the `isSet` could also be optional but we'll let it as is.